tests/AutoReview/ComposerTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\tests\AutoReview;
+
+use PhpCsFixer\Console\Application;
+use PhpCsFixer\Tests\TestCase;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ *
+ * @coversNothing
+ * @group auto-review
+ */
+final class ComposerTest extends TestCase
+{
+    public function testBranchAlias()
+    {
+        $composerJson = json_decode(file_get_contents(__DIR__.'/../../composer.json'), true);
+
+        if (!isset($composerJson['extra']['branch-alias'])) {
+            $this->assertTrue(true); // composer.json doesn't contain branch alias, all good!
```

Comment:
? maybe better to just increase the assert count and not an always pass assert?

Reply:
I was looking for that at night, could you please remind me the method ?

Reply:
`addToAssertionCount`

Reply:
That's why I couldn't find it :D 
```
ker@dus:~/github/PHP-CS-Fixer λ git grep addToAssertionCount
ker@dus:~/github/PHP-CS-Fixer λ 
```